### PR TITLE
update edgemesh integration doc

### PR DESCRIPTION
**Pull request by vincentgoat**
Signed-off-by: vincentgoat <linguohui1@huawei.com>

* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Bug fix

https://github.com/kubeedge/kubeedge/issues/4334

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Comment by vincentgoat** [comment]
cc @fisherxu

File: content/en/docs/advanced/edgemesh.md [hunk 1]
```diff
@@ -9,44 +9,9 @@ title: Edge connection with EdgeMesh
 toc: true
 type: docs
 ---
-In case network issue between cloud and edge side, we integrate EdgeMesh to support DNS visit at any time.
+In case of network issues between edge nodes, we integrate EdgeMesh to support the service discovery mechanism.
```
Comment (fisherxu) [inline content/en/docs/advanced/edgemesh.md:12]:
```suggestion
EdgeMesh as the data plane component of the KubeEdge cluster, provides simple service discovery and traffic proxy functions for applications, thereby shielding the complex network structure in edge scenarios.
```

File: content/en/docs/advanced/edgemesh.md [hunk 2]
```diff
@@ -9,44 +9,9 @@ title: Edge connection with EdgeMesh
...
-$ sudo sysctl -p
-```
-
-Then check it:
-
-```
-$ sudo sysctl -p | grep ip_forward
-net.ipv4.ip_forward = 1
-```
+Check more details at [edgemesh website](https://edgemesh.netlify.app) to deploy edgemesh application.
```
Comment (fisherxu) [inline content/en/docs/advanced/edgemesh.md:14]:
```suggestion
Check more details at [edgemesh website](https://edgemesh.netlify.app) to use EdgeMesh.
```

Comment (fisherxu) [reply content/en/docs/advanced/edgemesh.md:14]: And all the content below can be deleted I think? users can ge them from  https://edgemesh.netlify.app

**Pull request closed without merging**